### DISCUSSION TITLE
API get guild members

### DISCUSSION THREAD

**Discussion by pikachub2005** [discordjs/discord.js#7685] [Q&A] [answered]
I'm aware this doesn't have to do with discord.js, but I'd like to know how to get this.
https://discord.com/api/v10/guilds/guildID/members says I'm missing access, and the discord developer docs say it's a privileged intent. But you're still able to see other members on discord so idk how this works

**Comment by advaith1** [comment, marked as answer]
You need to enable the Server Members Intent in the Bot tab in the dev portal.

For help with the API go to https://discord.gg/discord-developers, don't create GitHub discussions in random repos.

**Reply by pikachub2005** [reply]
> You need to enable the Server Members Intent in the Bot tab in the dev portal.

I'm not using a bot, I'm using my own account with the api. That's why I said it's not related to discord.js. (I'm not automating anything if you're worried about that)

**Reply by monbrey** [reply]
> > You need to enable the Server Members Intent in the Bot tab in the dev portal.
> 
> I'm not using a bot, I'm using my own account with the api. That's why I said it's not related to discord.js. (I'm not automating anything if you're worried about that)

Yes you are. You're using the bot API with a user account token which violates the terms of service.

**Reply by pikachub2005** [reply]
> Yes you are. You're using the bot API with a user account token which violates the terms of service.

Isn't the same API used by used by bots and users? Eh I guess I won't mess with anything if it's against the ToS.
